snowflake/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 5.1.1 / 2023-11-15
+
+***Added***:
+
+* Added `pipe_schema` and `pipe_catalog` tags to snowpipe metrics. ([#16222](https://github.com/DataDog/integrations-core/pull/16222))
```

Comment:
@yogevyuval would you mind undoing this change, updating your `ddev` to the latest version and running `ddev release changelog new`? We recently reworked the way we generate changelogs.

Reply:
@iliakur Done. Can you take a look again?
Also - anyone can help with validating and tests?